src/services/MapMgr.ts

```diff
@@ -115,7 +115,9 @@ export class MapMgr {
     const res = await fetch(`${RADAR_URL}/obj/${mapType}/${mapName}/${hashId}/ai_groups`);
     return parse(res);
   }
-
+  getRegionFromPoint(layer: string, point: Point): Promise<string[]> {
```

Comment:
nit: there should be empty lines before and after the function to improve readability

Reply:
Done